rvps/src/bin/server/mod.rs

```diff
@@ -16,11 +16,11 @@ use crate::rvps_api::{
 };
 
 pub struct RVPSServer {
-    rvps: Arc<Mutex<Core>>,
+    rvps: Arc<Mutex<Rvps>>,
```

Comment:
I am considering change the `Mutex` here to `RwLock`. Let's do it in a seperate PR.